test/test_memcached_store.rb

```diff
@@ -379,20 +379,20 @@ def test_retains_encoding
   def test_initialize_accepts_a_list_of_servers_in_options
     options = { servers: ["localhost:21211"] }
     cache = ActiveSupport::Cache.lookup_store(:memcached_store, options)
-    assert_equal 21211, cache.instance_variable_get(:@data).servers.first.port
+    assert_equal ["localhost:21211:8"], cache.instance_variable_get(:@data).servers
```

Comment:
That's the (default) weight that `Memcached` adds to each address.
If you're asking if we should _not care_ about what the default value is... YES! I'll regex against these or something (i.e. not tie the weight to the assert)